sample/src/com/vungle/publisherSampleAndroid/MainActivity.java

```diff
@@ -101,6 +105,34 @@ public void run() {
 		}
 	};
 
+	private final EventListener vungleSecondListener = new EventListener() {
```

Comment:
Let's add a comment hear noting that the SDK allows for multiple event listeners to be attached. This secondary event listener is only going to print some logs for now, but it could be used to Pause music, update a badge icon, etc.